AK/Utf8View.cpp

```diff
@@ -274,6 +274,9 @@ u32 Utf8CodePointIterator::operator*() const
         code_point_value_so_far |= m_ptr[offset] & 63;
     }
 
+    if (code_point_value_so_far > 0x10FFFF) {
+        return 0xFFFD;
```

Comment:
No need for curlys with single line blocks

Reply:
Ah, right. Personally I like keeping the curlys always, but I see that's not how we do it here.

Come to think about it: The other parts of the code do `dbgln_if`, so it would be *even more* in keeping the style to add a `dbgln_if`-statement, which then also justifies the curlys, so that's what I'll do. What do you think?

Reply:
sure, sounds good